templates/example_flowsom_clustering.ipynb

```diff
@@ -18,6 +18,7 @@
    "outputs": [],
```

Comment:
<p>We can remove this now that we set a seed by default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>